packages/wdio-cli/src/constants.ts

```diff
@@ -431,8 +432,18 @@ export const QUESTIONNAIRE = [{
     // @ts-ignore
     default: [SUPPORTED_PACKAGES.service.find(
         /* istanbul ignore next */
-        ({ name }) => name === 'chromedriver').value
-    ],
+        ({ name }) => name === 'browserstack').value],
+    when: /* istanbul ignore next */ (answers: Questionnair) => answers.backend === 'In the cloud using Browserstack',
+    validate: /* istanbul ignore next */ (answers: string[]) => validateServiceAnswers(answers)
+}, {
+    type: 'checkbox',
+    name: 'services',
+    message: 'Do you want to add a service to your test setup?',
```

Comment:
I think we should be a bit more specific here
```suggestion
    message: 'Do you want to add the Browser Stack service to your test setup?',
```

Let's also add the same for Sauce Labs to stay feature compliant.

Reply:
Hi @christian-bromann,
I was thinking if we could include `browserstack-service` by default if the user selects "BrowserStack", and show the list of other services (without BrowserStack) to choose from.

Thought process behind this is that the browserstack-service improves the integration and there isn't a use-case where somebody would want to select No.

Wdyt?

Reply:
I am happy having the BS service pre-selected (e.g. instead of chromedriver) but don't think it is a good idea to hide the service entry.

Reply:
Okay, we have kept the BrowserStack service selected by default, and just moved it to the top of the list.